### PR TITLE
Adding UNION, INTERSECT and MINUS. Need to fix map on returned Result…

### PR DESCRIPTION
**Pull request by FCO**
…Seq. Related to #88

### CODE REVIEW

File: lib/Red/ResultSeq/Iterator.pm6 [hunk 1]
```diff
@@ -28,7 +28,12 @@ method pull-one {
     my $obj = $!of.new: |(%($data).kv
         .map(-> $k, $v {
             do with $v {
-                $k => try { $!driver.inflate(%cols{$k}.inflate.($v), :to($!of."$k"().attr.type)) } // %cols{$k}.inflate.($v)
+                $k => try {
+                    $!driver.inflate(
+                        %cols{$k}.inflate.($v),
+                        :to($!of."$k"().attr.type)
+                    )
+                } // (%cols{$k} ?? %cols{$k}.inflate.($v) !! $v)
```
Comment (jonathanstowe) [inline lib/Red/ResultSeq/Iterator.pm6]: :+1:

**Comment by FCO** [comment]
the tests are passing on my machine... Idk why it isn't on travis...

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,10 @@
...
-        key_url: 'http://keyserver.ubuntu.com/pks/lookup?search=0x379CE192D401AB61&op=get'
-    packages:
-      - rakudo-pkg
+language: perl6
+perl6:
+  - latest
 install:
+  - rakudobrew build-zef
   - zef install Test::META
   - zef install --deps-only --/test .
```
Comment (jonathanstowe) [inline .travis.yml]: You may find that this doesn't install any ```test-depends``` if there are any.  Just something that might catch you out.

**Comment by FCO** [comment]
Would you mind to test it on your machine? It’s working on mine... :(

**Comment by jonathanstowe** [comment]
Yeah strange, it works fine here too.   Let me have another look at that.

**Comment by FCO** [comment]
That’s why I’ve changed it to use Rakudo brew, to use the last commit, as I do at home....

**Comment by FCO** [comment]
Sorry to not doing anything for Red... I’m going to move shortly and I’m out of time to do any other thing

**Comment by jonathanstowe** [comment]
The above passes on a completely new Rakudo so :+1:

**Comment by jonathanstowe** [comment]
Well I literally just built a brand new Rakudo and ran the ```zef install .  --debug ``` just as it is on travis and it passes, so I'm confused.  Can't think how to debug this.

**Comment by FCO** [comment]
I have the same problem! I don’t know what to do...

**Comment by jonathanstowe** [comment]
The only remaining thing is the dependencies.  I'll try again with a completely new DBDish and see.

**Comment by jonathanstowe** [comment]
Well even completely fresh DBIish, zef, rakudo, Uncle Tom Cobbley and all and it still doesn't fail here.

Is it possible that the sqlite version is somehow failing in some way that is being hidden by the try in the ```pull-one``` ?

**Pull request merged**